snippets/logging-create-secret-snip.adoc

```diff
@@ -0,0 +1,24 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+You can create a secret in the directory that contains your certificate and key files by using the following command:
+[subs="+quotes"]
+[source,terminal]
+----
+$ oc create secret generic -n openshift-logging <my-secret> \
+ --from-file=tls.key=<your_key_file>
+ --from-file=tls.crt=<your_crt_file>
+ --from-file=ca-bundle.crt=<your_bundle_file>
+ --from-literal=username=<your_username>
+ --from-literal=password=<your_password>
+----
+
+[NOTE]
+====
+Generic or opaque secrets are recommended for best results.
```

Comment:
Nitpick: I'd make the note action-oriented ("use generic or opaque...") instead of using the passive voice.